plugins/module_utils/vmware.py

```diff
@@ -50,6 +50,64 @@ def __init__(self, *args, **kwargs):
         super(TaskError, self).__init__(*args, **kwargs)
 
 
+def is_datastore_valid(datastore_obj=None):
```

Comment:
Please check https://github.com/ansible-collections/community.vmware/pull/476 as I already did refactoring there (it was required for that fix).